index.bs

```diff
@@ -510,14 +510,14 @@ spec:css-syntax-3;
 
   <h5 id="algorithm-store-cred" algorithm>`[[Store]]` internal method</h5>
     <dfn for="Credential" method export>\[[Store]](credential, sameOriginWithAncestors)</dfn>
-    is called [=in parallel=] with a {{Credential}}, and a boolean which is true iff the caller's
+    is called [=in parallel=] with a {{Credential}}, and a boolean which is true if the caller's
```

Comment:
"iff" is a shorthand for "if and only if", so it is not a typo.

Reply:
Ah apologies didn't know that. Is that something defined somewhere that can be linked to or just "common knowledge"?

Reply:
I realised it's used elsewhere so will make a separate issue to address that.

Reply:
Link to that issue: https://github.com/w3c/webappsec-credential-management/issues/222